File: c/chuong_01/bai014.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    float a, b, c, d;
    cin >> a >> b >> c >> d;
    float tb = (a + b + c * 2 + d * 3) / 7;
    if (tb < 5) {
        cout << "YEU\n";
    }
    else if (tb < 6.5) {
        cout << "TRUNG BINH\n";
    }
    else if (tb < 8) {
        cout << "KHA\n";
    }
    else cout "GIOI\n";
    return 0;
}

File: python/chuong_01/bai014.py
def main():
    a, b, c, d = map(float, input().split())
    tb = (a + b + c * 2 + d * 3) / 7
    if tb < 5:
        print("YEU")
    elif tb < 6.5:
        print("TRUNG BINH")
    elif tb < 8:
        print("KHA")
    else:
        print("GIOI")

main()
